release/k8s_tests/run_gcs_ft_on_k8s.py

```diff
@@ -401,6 +403,10 @@ def main():
             for p in procs:
                 p.kill()
             print("==== Cleanup done ===")
+
+        if exception:
+            raise exception
```

Comment:
mmm what is happening here?

Reply:
We reraise the exception after cleanup - this test always exited successfully despite not actually working, as the exception was swallowed

Reply:
Why don't we raise it in row 392?

Reply:
I didn't want to modify the flow of the script (cleanup) - @sihanwang41 will be rewriting the test anyway

Reply:
@sihanwang41 , where are we tracking the rewrite?